src/components/RichText/__tests__/RichText.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import { render } from "@testing-library/vue";
+import RichText from "./../";
+
+describe("components/RichText", () => {
+  it("checks if the content will be passed in", () => {
+    const content = "test";
+    const { getByText } = render(RichText, {
+      props: {
+        content: content,
+      },
+    });
+    const contentTest = getByText("test");
+    expect(contentTest).toBeTruthy;
+  });
+  it("checks if inline works", () => {
+    const content = "test";
+    const inline = true;
+    const { getByTestId } = render(RichText, {
+      props: {
+        content: content,
+        inline: inline,
+      },
```

Comment:
Please use the object initializer shorthand:
`props: { content, inline },`

Reply:
done